### DISCUSSION TITLE
Token expiration date for one year is too restrictive

### DISCUSSION THREAD

**Discussion by DimRochette** [community/community#85668] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Today we can't set token to expire in more than on year.
This is OK 

When my token will soon expire I have warnings and can regenerate the token (great UX from website). 
But I can't generate te token to the same date before the expiration is reached. 
So either I move the date each year by a few day earlier, or I've to move my token to 6 month to keep things predictably consistent. 

I would love to be able to regenerate the token to the same expiration day as the previous one but one year later.

**Comment by LuksJobs** [comment]
You're right, it's not ideal that you can't regenerate the token exactly one year ahead of its current expiration date. Here are some solutions you can consider depending on your priorities and the website's capabilities:

Manually adjust the date:

- Move the date early: As you mentioned, you can manually move the expiration date a few days earlier each year before it expires. This is simple but requires you to remember to do it annually.

- Set it to 6 months: Choose a fixed date 6 months ahead for all future renewals. This keeps things predictable but may not align with your desired annual cycle.

Ultimately, the best solution depends on your specific needs and the website's capabilities. Weigh the pros and cons of each option and choose the one that works best for you.

I hope this helps! Let me know if you have any other questions.
